lsif-java/src/main/scala/com/sourcegraph/lsif_java/buildtools/GradleJavaCompiler.scala

```diff
@@ -98,6 +98,16 @@ case class GradleJavaCompiler(languageVersion: String, javacPath: Path) {
       )
       .toFile
       .setExecutable(true)
+    // for compileKotlin when using jvm toolchains
+    val libPath = dir.resolve("lib")
+    Files
+      .walk(javacPath.getParent.getParent.resolve("lib"))
+      .forEach(source => {
+        val destination = libPath.resolve(
+          javacPath.getParent.getParent.resolve("lib").relativize(source)
```

Comment:
Maybe hoist out `javacPath.getParent.getParent.resolve("lib")` and give it a name?